cloudfront/wellcomecollection.org/terraform/locals.tf

```diff
@@ -1,8 +1,5 @@
 
 locals {
-  # identity_zone_name_servers = data.terraform_remote_state.identity.outputs.identity_zone_name_servers
```

Comment:
This is no longer provided by the identity stack and has no usages, so removing here to get a clean plan/apply - it doesn't change any resources here.